project/Dependencies.scala

```diff
@@ -42,7 +42,7 @@ object Dependencies {
 
   lazy val db = Seq(
     "com.h2database" % "h2-mvstore" % "1.4.196",
-    "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8"
+    "org.openlabtesting.leveldbjni" % "leveldbjni-all" % "1.8"
```

Comment:
I think we still need to keep the previous package, but in the next release, we can build a version with this package and remind the risk to all users.

Reply:
Can you explain the risk to me? I'm just not seeing it

Reply:
I think I have explained to you about it before. And support about this is also my idea and gives an issue about this, the official release has no such functionality but in their master branch they supported. If users trust this they can package the source code to their project and build their version. Users should take this risk by themselves. This is the same reason which I listed in the original issue. If you think there is no risk about using this, please give that the master branch code has the same level as the released one? And why it should choose the one listed here. @sunnyking @zl730 please check this and all other updates in this PR.

Reply:
For the time being, I will break this PR down per your request.

```
If you think there is no risk about using this, please give that the master branch code has the same level as the released one?
```

I am not sure what you mean here.

Reply:
> Can you explain the risk to me? I'm just not seeing it.
> 
> I did review the code in the new lib per your request.
> 
> The current lib was last updated in 2013:
> 
> https://mvnrepository.com/artifact/org.fusesource.leveldbjni/leveldbjni-all
> 
> The one I'm proposing gets routine updates:
> 
> https://mvnrepository.com/artifact/org.opendaylight.odlparent/leveldbjni-all

The one you mentioned in the second link is not the one you cited in this pr~

Reply:
1. The master branch of `leveldbjni` is not the release one, which can not say it has the same security level as the release one.
2. All users can package the code and submit their packages. We have no evidence that the one cited here built from the same code of that master branch.
3. Check the usage list, the previous one is well used and cited by many projects, but this one not~
4. I also give some suggestions to users who need this support should build the jar file themselves by switch this package before. If we need to cite a new package we need to check the master branch of `leveldbjni` ourselves and make sure it works or build and package it ourselves, since it is not from the official release supported by their developers and backed by the source code.